docs/privacy-policy.rst

```diff
@@ -102,7 +102,7 @@ Why do we collect this?
 - We limit our use of your User Personal Information to the purposes listed in this Privacy Statement.
   If we need to use your User Personal Information for other purposes, we will ask your permission first.
   You can always see what information we have in your
-  `user account <https://readthedocs.org/accounts/edit/>`_.
+  `user account <https://readthedocs.org/accounts/edit/>`__.
```

Comment:
Using anonymous refs is recommended (lost the link for this statement :/, but it was on a sphinx issue)